Diagnostics/start-windows-build.ps1

```diff
@@ -0,0 +1,482 @@
+Param(
```

Comment:
I think that this change is added to the PR https://github.com/Microsoft/mesos-jenkins/pull/37 and should not be included here.

Reply:
I will remove it